wiki/Tournaments/TWC/2021/zh.md

```diff
@@ -3,6 +3,7 @@ tags:
   - TWC
   - TWC2021
   - 世界杯
+outdated: true
```

Comment:
comment would be helpful

Reply:
Since the translation hasn't been updated since the round of 16, an `outdated_since` might be more helpful:
```suggestion
outdated: true
outdated_since: fd12864fe0d9d5c4e43f09e681be0c1ad040e200
```

If that isn't desirable, it can just be removed
```suggestion
```

Reply:
`outdated_since` is ok